eng/pipelines/templates/scripts/verify-path-length.yml

```diff
@@ -22,16 +25,17 @@ steps:
         print('Analyzing length of paths...')
         for root, dirs, files in os.walk('{0}'.format(source_directory)):
             for file in files:
-                file_path = os.path.join(root, file)
-                if (len(file_path) > longest_file_path_length):
-                    longest_file_path_length = len(file_path)
+                file_path = os.path.relpath(os.path.join(root, file), source_directory)
+                if ((len(file_path) + ${{ parameters.BasePathLength }}) > longest_file_path_length):
+                    longest_file_path_length = len(file_path) + ${{ parameters.BasePathLength }}
```

Comment:
Shouldn't you move the error messages outside of the loops so that you can actually calculate the longest path? Or is it your intention to fail on the first one that is too long?

Reply:
You are right, am fixing this now.

Reply:
I updated the script with PR https://github.com/Azure/azure-sdk-tools/pull/316/files